aasemble/django/apps/api/tests.py

```diff
@@ -302,8 +302,8 @@ def test_create_source_incorrect_auth_fails_401(self):
 
         self.assertEquals(response.status_code, 401)
 
-    def test_create_source(self):
-        authenticate(self.client, 'eric')
+    def test_create_source(self, default_user='eric'):
```

Comment:
Can we call this argument just `user`, rather than `default_user`? Eric is the default user. What you're allowing callers to pass in is a value that *overrides* the default user.